.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: "❓ IRC"
-    url: https://web.libera.chat/
-    about: Please ask and answer questions here, (#jenkins-release on libera.chat).
+  - name: "💬 Chat"
+    url: https://matrix.to/#/#jenkins-release:libera.chat
+    about: Please ask and answer questions here.
```

```suggestion
    about: Please ask and answer questions here.
  - name: "❓ IRC"
    url: https://web.libera.chat/
    about: Click here to access IRC from a browser (#jenkins-release on libera.chat).
```
Keep the previous IRC link but at the end.